FinniversKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FinniversKit'
-  s.version      = '71.0.0'
+  s.version      = '71.0.1'
```

Comment:
Better to follow this: https://github.com/finn-no/FinniversKit/blob/master/RELEASING.md

And isn’t it a minor version change since it adds functionality instead of fixing bug?

Reply:
I will do! 71.1.0 then?

Reply:
Yes, or whatever the script tells you is next minor when you have merged 😄

Reply:
great, AI taking over 💯